kubevirtmng/ui/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang = "en">
+
+   <head>
+      <meta charset = "UTF-8">
+      <title>Disks</title>
+      <!-- Latest compiled and minified CSS -->
+	    <!-- PatternFly Styles -->
+
+      <!-- jQuery -->
```

Comment:
In general webpack can suck all of this in for you so you don't need all of these external <script> tags (and you're already using webpack, so these are possibly/probably doing nothing)

Webpack can also pull in css as part of loaders, with:

{
  test: /\.css$/,
  loader: "style-loader!css-loader"
}

(which need to be installed with npm beforehand, but still may be nicer.